website/websitecore/static/websitecore/css/landing_page.css

```diff
@@ -0,0 +1,522 @@
+
+.footer-buttom {
+    display: flex;
+    align-items: center;
+    gap: 97px;
+    padding: 20px;
+}
+
+.footer-buttom img {
+    margin-left: 20px;
+    height: 35PX;
+}
+
+.footer-buttom p {
+    font-size: 10px;
+}
+
+ul {
+    list-style-type: none;
+}
+
+ol {
+    list-style-type: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+h3 {
+    color: white;
+    position: absolute;
+    font-family: 'Lexend', sans-serif;
+}
+@media screen and (max-width: 800px) {
+    .service-grid, .study-grid{
+        grid-template-columns: 1fr;
+    }
+    .case-study h3{
+        margin-left: -160px;
+        font-size: 50px;
+    }
+    .footer-top{
+        grid-template-columns: 1fr 1fr;
+    }
+    .footer-buttom{
+        flex-direction: column;
+        margin-left: -50px;
+        gap: 5px;
+    }
+    .services h3{
+        margin-left: -150px;
+    }
+    .navbar{
+        display: none;
+    }
+    .about-content p{
+        font-size: 14px;
+        margin-left: 10px;
+        margin-right: -290px;
+    }
+    .about-us{
+        margin-left: -300px;
+        height: 100%;
+    }
+    .about-content img{
+        margin-left: 160px;
+    }
+    .phone-img{
+        display: block;
+        position: absolute;
+        background: white;
+        border-radius: 40px;
+        height: 140px;
+        width: 100%;
+        box-shadow: 10px 10px 5px rgba(0, 0, 0, 0.3);
+    }
+    .phone-img img{
+        height: 100px;
+        width: 140px;
+        margin-left: 30px;
+        margin-top: 15px;
+    }
+    .navbar ul a li:hover+.sub-navbar {
+        display: block;
+        background: white;
+        position: absolute;
+        font-size: 9px;
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr 1fr;
+        margin-left: -582px;
+        margin-top: 26px;
+        box-shadow: 10px 10px 5px rgba(0, 0, 0, 0.3);
+    }
+
+    .navbar ul a li:hover+.sub-navbar .trak-grid img {
+        height: 70px;
+    }
+
+    .navbar ul a li:hover+.sub-navbar .trak-grid {
+        gap: 20px;
+    }
+
+    .navbar ul a li:hover+.sub-navbar h4 {
+        font-size: 16px;
+    }
+
+    .navbar ul a li:hover {
+        text-decoration: underline;
+    }
+
+    .sub-navbar-2 {
+        background: white;
+        padding: 20px;
+        width: 1223px;
+        gap: 10px;
+        display: grid;
+        display: none;
+        border-radius: 0 0 10px 10px;
+        margin-top: 26px;
+    }
+
+    .trak-2-grid {
+        display: grid;
+        grid-template-columns: 100px 100px 100px;
+    }
+
+    .navbar ul a li:hover+.sub-navbar-2 {
+        display: block;
+        background: white;
+        box-shadow: 10px 10px 5px rgba(0, 0, 0, 0.3);
+        position: absolute;
+        font-size: 9px;
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr;
+        margin-left: -817px;
+    }
+
+    .navbar ul a li:hover+.sub-navbar-2 .trak-2-grid img {
+        height: 70px;
+    }
+
+    .navbar ul a li:hover+.sub-navbar-2 .trak-2-grid {
```

Comment:
This looks too complicated. 
Look for a simpler way to do it, maybe using classes will be better